spec/features/homepage_text_spec.rb

```diff
@@ -44,11 +44,6 @@
     end
   end
 
-  xscenario "click 'Register a new application' link" do
-    click_link I18n.t('links.register_a_new_application')
-    expect(page).to have_content 'Register a new application'
-  end
-
   xscenario "click 'view' link" do
```

Comment:
Let's get rid of this one too, since it's also talking about applications and moved to pending with `xscenario`